### ISSUE TITLE
Lookup user country not working

### ISSUE THREAD

**Issue by shahzada9t5** [jackocnr/intl-tel-input#1445]
Hi, I want to use Lookup user country, but I am getting cors error even on Demo Page.  Please guide how to resolve this...

<img width="1475" alt="image" src="https://github.com/jackocnr/intl-tel-input/assets/121219581/daff1b53-f5e3-47ef-814c-254c49be6c29">

**Comment by jackocnr** [comment]
I have no idea about this I'm afraid. It could be an issue with your Chrome settings? Maybe try another browser?

**Comment by shahzada9t5** [comment]
yes, i checked with multiple browser but issue still same. i added intl tel input into my project but still same results. cors issue...

**Comment by jackocnr** [comment]
What device/OS/version are you on? And please can you list all browsers/versions that you have tried.

**Comment by jackocnr** [comment]
Closing due to inactivity.

**Comment by ipapi-co** [comment]
Leaving this remark here in case it helps other users : https://github.com/jackocnr/intl-tel-input/issues/1547#issuecomment-2058888180
